src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -355,28 +347,23 @@ def killJob(self, jobIDList):
   def getCEStatus(self):
     """ Method to return information on running and pending jobs.
     """
+    if not self.useLocalSchedd:
+      return S_ERROR('Failed to get CE status: method not supported when using a remote schedd.')
```

Comment:
Well, in the case we cannot get the status of a CE, we can disable the `QueryCEFlag` (which is already disabled by default) in the CE: the Site Director interrogates `PilotAgentsDB` instead of the CE in this case. Am I missing something here?

see: https://github.com/DIRACGrid/DIRAC/blob/integration/src/DIRAC/WorkloadManagementSystem/Agent/SiteDirector.py#L841-L878

From the Site Director we could also try to get information from the CE first, and in the case it does not work, then we interrogate the `PilotAgentsDB`.

Reply:
The CE object should be able to work alone. The fact that its main client is the SiteDirector, and the SiteDirector has this flag, and so on, does not change matters.

Reply:
If the CE object should be able to work alone, then calling `PilotAgentsDB` to get the CE status will return a false information: it will not be able to take into account the pilots/jobs we sent with it.
But ok, if you think that it is simpler, then I will keep the dependency with `PilotManager`.

Reply:
Yes, please just change the dependency : from PilotAgentsDB to PilotManager.